kvmagent/kvmagent/test/utils/my_debugger.py

```
@@ -0,0 +1,366 @@
+        ("ls -la", "Directory listing"),
+        ("which python", "Python executable location"),
+        ("python -c \"import sys; print('Python path:', sys.path)\"", "Python path from interpreter"),
+        ("find /root/.zguest -name 'zstacklib' -type d 2>/dev/null | head -5", "Find zstacklib directories"),
+        ("find /usr/local -name 'zstacklib' -type d 2>/dev/null | head -5", "Find zstacklib in /usr/local")
+    ]
+    
+    for cmd, description in commands:
+        print("\n" + description + ":")
+        print("  Command: " + cmd)
+        try:
+            result = subprocess.Popen(
```

Ruff (0.15.9)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)